static/js/hotkey.js

```diff
@@ -132,9 +133,11 @@ const keypress_mappings = {
     74: {name: "vim_page_down", message_view_only: true}, // 'J'
     75: {name: "vim_page_up", message_view_only: true}, // 'K'
     77: {name: "toggle_topic_mute", message_view_only: true}, // 'M'
+    78: {name: "resume_reading", message_view_only: true}, // 'N'
```

Comment:
This `N` key doesn't do the right thing and hasn't been discussed; removing it for now.